atoms/alert/index.html

```diff
@@ -0,0 +1,113 @@
+      possible designs for different use cases:
+      <ul>
+        <li>
+          <h3 class="tui__h-3 tui__m-tb-lg">Simple Alerts</h3>
+          <ul class="tui__bw-sm tui__pd-md tuid__li--component">
+            <li>
+              <div class="tui__alert--success"><p class="tui__text-xs">Success Text</p></div>
+            </li>
+            <li>
+              <div class="tui__alert--info"><p class="tui__text-xs">Info Text</p></div>
+            </li>
+            <li>
+              <div class="tui__alert--warning"><p class="tui__text-xs">Warning Text</p></div>
+            </li>
+            <li>
+              <div class="tui__alert--error"><p class="tui__text-xs">Error Text</p></div>
+            </li>
+          </ul>
+          <div class="preformat tui__m-tb-lg">
+            <pre id><code class="language-markup">
+              
+    <div class="tui__alert--success">
+      <p class="tui__text-xs">Success Text</p>
+    </div>
+
+    <div class="tui__alert--info">
+      <p class="tui__text-xs">Info Text</p>
+    </div>
+
+    <div class="tui__alert--warning">
+      <p class="tui__text-xs">Warning Text</p>
+    </div>
+
+    <div class="tui__alert--error">
+      <p class="tui__text-xs">Error Text</p>
+    </div>
+
+              </code></pre>
+          </div>
+        </li>
+        <h3 class="tui__h-3 tui__m-tb-lg">Alerts heads</h3>
+          <ul class="tui__bw-sm tui__pd-md tuid__li--component">
+            <li>
+              <div class="tui__alert--success"> <h4 class="tui__h-4">Success</h4> <p class="tui__text-xs">Success Text Success Text Success Text</p></div>
+            </li>
+            <li>
+              <div class="tui__alert--info"> <h4 class="tui__h-4">Info</h4> <p class="tui__text-xs">Info Text Info Text</p></div>
+            </li>
+            <li>
+              <div class="tui__alert--warning"> <h4 class="tui__h-4">Warning</h4> <p class="tui__text-xs">Warning Text Warning Text</p></div>
+            </li>
+            <li>
+              <div class="tui__alert--error"> <h4 class="tui__h-4">Error</h4> <p class="tui__text-xs">Error Text Error Text</p></div>
+            </li>
+          </ul>
+          <div class="preformat tui__m-tb-lg">
+            <pre id><code class="language-markup">
+              
+    <div class="tui__alert--success">
+      <h4 class="tui__h-4">Success</h4>
+      <p class="tui__text-xs">Success Text Success Text Success Text</p>
+    </div>
+
+    <div class="tui__alert--info"> 
+      <h4 class="tui__h-4">Info</h4>
+      <p class="tui__text-xs">Info Text Info Text</p>
+    </div>
+
+    <div class="tui__alert--warning">
+      <h4 class="tui__h-4">Warning</h4>
+      <p class="tui__text-xs">Warning Text Warning Text</p>
+    </div>
+
+    <div class="tui__alert--error">
+      <h4 class="tui__h-4">Error</h4>
+      <p class="tui__text-xs">Error Text Error Text</p>
+    </div>
+
+              </code></pre>
+          </div>
+        </li>
+      </ul>
+    </main>
```

Comment:
Here also

Reply:
@romabulani Its was mentioned as part of Prism library which I am using to highlight, "that I should stick these two tags together". This is the reason why it is there. Otherwise, it may fail to render in some of the browsers.